### ISSUE TITLE
NLC classifier creation with training data from memory

### ISSUE THREAD

**Issue by jtviegas** [watson-developer-cloud/java-sdk#501]
It would be great if the NLC library could provide a method to create a classifier with an InputStream, additionally to the existing one with the File param: 
```java
public ServiceCall<Classifier> createClassifier(final String name, final String language, final File trainingData)
```

...so that the library client doesn't have to access the server file system.

**Comment by germanattanasio** [comment]
This is a great idea, I will work on it after holidays.

**Comment by zverbatim** [comment]
curious if any updates on this?

**Comment by germanattanasio** [comment]
We are going to work on this soon. I would saw 2 weeks

**Comment by germanattanasio** [comment]
This should be fixed in `v5.0.1` You can now use an `InputStream`. Make sure specify the file name and contentType too
